doc/introduction/importing_workflows.rst

```diff
@@ -0,0 +1,231 @@
+.. role:: html(raw)
+   :format: html
+
+.. _intro_ref_importing_circuits:
+
+Importing workflows
+===================
+
+PennyLane supports importing quantum circuits and operations that were
+constructed using another framework. This includes circuits defined using `Qiskit <https://www.ibm.com/quantum/qiskit>`__,
+`OpenQASM <https://arxiv.org/abs/1707.03429>`_, and `Quil
+<https://docs.rigetti.com/qcs/guides/quil>`_.
+
+.. note::
+
+    To import a quantum circuit defined using a particular framework, you will need to install the
+    corresponding PennyLane plugin for that framework. More information about PennyLane plugins is
+    available on the `plugins <https://pennylane.ai/plugins.html>`_ page.
+
+Importing quantum circuits
+--------------------------
+
+Importing quantum circuits can help you take advantage of PennyLane's various optimization,
+visualization, and interoperability features for existing circuits. For example, you can take a
+`QuantumCircuit <https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.QuantumCircuit>`__ from
+Qiskit, import it into PennyLane, and then apply a `circuit-cutting transform
+<https://pennylane.ai/qml/demos/tutorial_quantum_circuit_cutting/>`_ to reduce the number of qubits
+required to implement the circuit. You could also :ref:`compile <intro_ref_compile_circuits>` the
```

Comment:
```suggestion
required to implement the circuit. You could also :ref:`compile <intro_ref_compile_worklfows>` the
```
Would be good to point to the workflows compilation page [here](https://xanaduai-pennylane--5281.com.readthedocs.build/en/5281/introduction/compiling_workflows.html).

Note that the typo in the suggestion is accurate :laughing: